user/git-extras/template.py

```diff
@@ -0,0 +1,100 @@
+pkgname = "git-extras"
+pkgver = "7.1.0"
+pkgrel = 0
+depends = ["git", "bash"]
+pkgdesc = "Extra Git utilities"
+maintainer = "ttyyls <contact@behri.org>"
+license = "BSD-2-Clause"
+url = "https://github.com/tj/git-extras"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "e5c855361d2f1ec1be6ee601247153d9c8c04a221949b6ec3903b32fa736f542"
+
+
+def do_install(self):
+    for utility in [
+        "git-abort",
+        "git-alias",
+        "git-archive-file",
+        "git-authors",
+        "git-browse",
+        "git-browse-ci",
+        "git-brv",
+        "git-bulk",
+        "git-changelog",
+        "git-clear",
+        "git-clear-soft",
+        "git-coauthor",
+        "git-commits-since",
+        "git-contrib",
+        "git-count",
+        "git-cp",
+        "git-create-branch",
+        "git-delete-branch",
+        "git-delete-merged-branches",
+        "git-delete-squashed-branches",
+        "git-delete-submodule",
+        "git-delete-tag",
+        "git-delta",
+        "git-effort",
+        "git-extras",
+        "git-feature",
+        "git-force-clone",
+        "git-fork",
+        "git-fresh-branch",
+        "git-get",
+        "git-gh-pages",
+        "git-graft",
+        "git-guilt",
+        "git-ignore",
+        "git-ignore-io",
+        "git-info",
+        "git-local-commits",
+        "git-lock",
+        "git-locked",
+        "git-magic",
+        "git-merge-into",
+        "git-merge-repo",
+        "git-missing",
+        "git-mr",
+        "git-obliterate",
+        "git-paste",
+        "git-pr",
+        "git-psykorebase",
+        "git-pull-request",
+        "git-reauthor",
+        "git-rebase-patch",
+        "git-release",
+        "git-rename-branch",
+        "git-rename-remote",
+        "git-rename-tag",
+        "git-repl",
+        "git-reset-file",
+        "git-root",
+        "git-sed",
+        "git-setup",
+        "git-show-merged-branches",
+        "git-show-tree",
+        "git-show-unmerged-branches",
+        "git-squash",
+        "git-stamp",
+        "git-standup",
+        "git-summary",
+        "git-sync",
+        "git-touch",
+        "git-undo",
+        "git-unlock",
+        "git-utimes",
+    ]:
+        self.install_bin("bin/" + utility)
+        self.install_man("man/" + utility + ".1")
+
+    # utilities without a dedicated manpage
+    self.install_bin("bin/git-scp")
+    self.install_link("git-scp", "usr/bin/git-rscp")
```

Comment:
```suggestion
    self.install_bin("bin/*", glob=True)
    self.install_link("git-scp", "usr/bin/git-rscp")
    self.install_man("man/*.1", glob=True)
```

Reply:
well- you'd have to `self.rm` git-rscp first to replace it with a link, since the first install will deref it and install a dupe, i guess

Reply:
globbing causes git-rcsp to be selected anyway, which breaks the whole do_install step since hence the manual listing of utilities

Reply:
that would make sense if i didn't just say how to work around it

Reply:
mb I missed that

Reply:
or even better:  

```
    self.install_files("bin", "usr")
```

then it installs the link normally

Reply:
so the whole thing:  

```
def do_install(self):
    self.install_files("bin", "usr")
    self.install_man("man/*.1", glob=True)
    self.install_completion("etc/bash_completion.sh", "bash")
    self.install_completion("etc/git-extras-completion.zsh", "zsh")
    self.install_completion("etc/git-extras.fish", "fish")
    self.install_license("LICENSE")
```

Reply:
most definitely